<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>BOSC 2014 - Biopython Update</title>

		<meta name="description" content="Biopython Update 2014 (presented in BOSC 2014)">
		<meta name="author" content="Wibowo Arindrarto">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="css/reveal.min.css">
		<link rel="stylesheet" href="css/theme/default.css" id="theme">
		<link rel="stylesheet" href="css/custom.css">

		<!-- For syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- If the query includes 'print-pdf', include the PDF print sheet -->
		<script>
			if( window.location.search.match( /print-pdf/gi ) ) {
				var link = document.createElement( 'link' );
				link.rel = 'stylesheet';
				link.type = 'text/css';
				link.href = 'css/print/pdf.css';
				document.getElementsByTagName( 'head' )[0].appendChild( link );
			}
		</script>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">

				<section>
					<h1>Biopython Project Update</h1>
					<h3>Bioinformatics Open Source Conference
          <br />
          Boston, July 10th 2014</h3>
          <img style="border-radius: 10px; width: 35%;" src="img/biopython.jpg" />
					<p>
              <a href="http://twitter.com/_bow_">Wibowo Arindrarto</a> (Bow),
              <a href="http://twitter.com/pjacock">Peter Cock</a>,
              <a href="http://twitter.com/etalevich">Eric Talevich</a>,
              <br />and the Biopython contributors
					</p>
          <p style="margin-top: 1em; margin-bottom: 0; font-size: .8em;">
            <a href="https://twitter.com/_bow_">@_bow_</a>
            &middot;
            <a href="https://twitter.com/search?q=%23BOSC2014">#BOSC2014</a>
            &middot;
            <a href="https://twitter.com/search?q=%23ISMB2014">#ISMB2014</a>
          </p>
          <a href=http://creativecommons.org/licenses/by-sa/4.0/" title="Licensed under the Creative Commons Attribution-ShareAlike 4.0 Licence">
            <img style="border-radius: 5px;" src="img/cc-by-sa-4.0.png" />
          </a>

          <aside class="notes">
            Short self intro: contributed SeqIO format, then SearchIO in GSoC, now co-mentoring with Peter (got a job in between)
          </aside>
				</section>

				<section>
          <h1>The Introduction</h1>
          <ul>
            <li>Collection of modules for dealing with Biological data in Python</li>
            <li>Open source + freely available</li>
            <li>Multiple OSes + multiple Python implementations &amp; versions</li>
            <li>First commit in 1999</li>
          </ul>
				</section>

				<section>
          <h1><code>show_stats(Bio)</code></h1>
          <figure>
            <img style="border-radius: 10px; width: 70%;" src="img/commits.png" />
            <figcaption><a href="http://www.ohloh.net/p/biopython/analyses/latest/languages_summary"><small>via Ohloh (June 2014)</small></a></figcaption>
          </figure>
          <ul>
            <li>&gt;9,000 commits by &gt;100 contributors</li>
            <li>Since BOSC 2013: &gt;800 commits &amp; 17 new contributors</li>
          </ul>
				</section>

        <section>
          <h2>New contributors since BOSC 2013</h2>
          <div style="width: 70%; margin: auto;">
            <div style="float: left;">
              <ul>
                <li>Bertrand NÃ©ron</li>
                <li>Chris Mitchell</li>
                <li>Chunlei Wu</li>
                <li>Edward Liaw</li>
                <li>Gokcen Eraslan</li>
                <li>Josha Inglis</li>
                <li>Konstantin Tretyakov</li>
                <li>Manlio Calvi</li>
                <li>Markus Piotrowski</li>
              </ul>
            </div>
            <div style="float: right;">
              <ul>
                <li>Matsuyuki Shirota</li>
                <li>Melissa Gymrek</li>
                <li>Nigel Delaney</li>
                <li>Sergei Lebedev</li>
                <li>Vincent Davis</li>
                <li>Wayne Decatur</li>
                <li>Yanbo Ye</li>
                <li>Zheng Ruan</li>
              </ul>
            </div>
          </div>
        </section>

        <section>
          <h1>Last Time in BOSC</h1>
          <ul>
            <li>Peter Cock presented in Berlin</li>
            <li>Biopython 1.62 beta released (1.62 released shortly after)</li>
            <li>Two Google Summer of Code (GSoC) students (with NESCent)</li>
          </ul>
        </section>

        <section>
          <h1><code>Bio.CodonAlignment</code> <small>Biopython 1.64</small></h1>
          <ul>
            <li>GSoC 2013 project, now an experimental module</li>
            <li>Codon alignment analysis: objects &amp; analysis methods</li>
            <li><em>Mentors</em>: Eric Talevich &amp; Peter Cock</li>
          </ul>
        </section>

        <section>
          <h1><code>Bio.Phylo</code> enhancements <small>Biopython 1.64</small></h1>
          <ul>
            <li>GSoC 2013 project</li>
            <li>Improved phylogenetic analysis with <code>Bio.Phylo</code>: tree consensus, tree comparisons, etc.</li>
            <li><em>Mentors</em>: Mark Holder, Jeet Sukumaran, &amp; Eric Talevich</li>
          </ul>
        </section>

        <section>
          <h1>Other additions <br /><small>Biopython 1.62-1.65+</small></h1>
            <ul>
              <li><code>Bio.Applications</code>: <a href="http://cmpg.unibe.ch/software/fastsimcoal2/">fastsimcoal</a>, <a href="http://cmpg.unibe.ch/software/fastsimcoal2/">bwa</a>, and <a href="">samtools</a> support</li>
              <li><code>Bio.Entrez</code> DTD caching</li>
              <li><code>Bio.Restriction</code> enzyme list</li>
              <li><code>Bio.SearchIO</code>, <code>Bio.Phylo</code>, and <code>Bio.motifs</code> format support</li>
              <li style="margin-top: 1em;">More from yesterday's <a href="http://twitter.com/search?q=%23codefest">#codefest</a>: Brad's update tomorrow</li>
            </ul>
        </section>

        <section>
          <h1>Streamlined Codebase <small>Biopython 1.63</small></h1>
          <ul>
            <li>Single codebase supporting Python 2, Python 3, PyPy, and Jython 2.7</li>
            <li>Previously: <code>2to3.py</code></li>
            <li>Had to drop Python 2.5 (consequently Jython 2.5) support</li>
            <li>Documentation, tutorial, cookbook updated accordingly</li>
          </ul>
        </section>

        <section>
          <h1>Wider Availability</h1>
          <ul>
            <li>In the Galaxy ToolShed as a tool dependency</li>
            <li>Version 1.61 onwards</li>
          </ul>
          <p style="margin-top: 1em;">
            Dev repo: <a href="http://github.com/biopython/galaxy_packages">http://github.com/biopython/galaxy_packages</a>
          </p>
        </section>

        <section>
          <h1>GSoC 2014</h1>
          <div style="width: 80%; margin: auto;">
            <div style="float: left; width: 73%;">
              <ul>
                <li>Evan Parker, Chemistry PhD candidate
                <li>Addition of lazy-parsing support for <code>Bio.SeqIO</code></li>
                <li>Faster parsing for simple formats (e.g. FASTA) and complex, annotated formats (e.g. GenBank)</li>
                <li><em>Mentors</em>: Peter Cock &amp; Wibowo Arindrarto</li>
              </ul>
            </div>
            <div style="float: right;">
              <img src="img/evan.jpg" style="border-radius: 10px; z-index: 10;" height="200px;">
            </div>
            <div style="float: left;">
              <ul>
                <li style="margin-top: 1em;">Blog: <a href="http://blog.evanaparker.com">http://blog.evanaparker.com</a></li>
                <li>Dev repo: <a href="http://github.com/eparker05/biopython">http://github.com/eparker05/biopython</a></li>
              </ul>
            </div>
          </div>
        </section>

        <section>
          <h1>Acknowledgement</h1>
          <h5 style="margin-top: 20px; margin-bottom: 5px;">The Community</h5>
          <p>contributors, users, testers, bug reporters, everybody involved</p>
          <h5 style="margin-top: 30px; margin-bottom: 5px;">Infrastructure &amp; Other Support</h5>
          <div>
            <img src="img/obf.jpg" style="border-radius: 10px; margin-bottom: 0px;" height="80px" title="Open Bioinformatics Foundation" />
            <img src="img/google.jpg" style="border-radius: 10px; margin-bottom: 0px;" height="80px" title="Google" />
            <p>
            <img src="img/github.jpg" style="border-radius: 10px; margin-top: 5px;" height="80px" title="GitHub" />
            <img src="img/travis.jpg" style="border-radius: 10px; margin-top: 5px;" height="80px" title="Travis CI" />
          </div>
        </section>

        <section>
          <h1>I Thank ...</h1>
            <div>
              <img style="border-radius: 10px; margin-bottom: .1em; width: 25%;" src="img/lumc.jpg" />
              <p>
                <ul>Biopython lives in our:
                  <li>In-house scripts and pipelines</li>
                  <li>Released Python packages (available in the PyPI):
                    <ul>
                      <li><code><a href="http://pypi.python.org/pypi/kMer/0.2.0">kMer</a></code>: k-mer analysis toolkit</li>
                      <li><code><a href="http://humgenprojects.lumc.nl/trac/humgenprojects/wiki/TSSV">tssv</a></code>: short structural variation analysis<br /><small>Anvar et al. (2014) DOI:10.1093/bioinformatics/btu068</small></li>
                    </ul>
                  </li>
                  <li style="margin-top: -10px;">Mutalyzer web service: <a href="http://mutalyzer.nl/">http://mutalyzer.nl</a><br />an HGVS variant nomenclature checker
                  </li>
                </ul>
              </p>
            </div>
        </section>

        <section>
          <h1>Thank you!</h1>
          <p style="margin-top: 1em;">Check us out:
            <a href="http://biopython.org">http://biopython.org</a>
          </p>
          <p style="margin-top: 1em;">Code repository:
            <ul>
              <a href="http://github.com/biopython/biopython">http://github.com/biopython/biopython</a>
            </ul>
          </p>
          <p style="margin-top: 1em;">Mailing list:
            <ul>
              <li>General list: <a href="mailto:biopython@biopython.org">biopython@biopython.org</a></li>
              <li>Developers list: <a href="mailto:biopython-dev@biopython.org">biopython-dev@biopython.org</a></li>
            </ul>
          </p>
          <p style="margin-top: 1em;">This presentation:
            <a href="http://speakerdeck.com/bow/2014-bosc-biopython">http://speakerdeck.com/bow/2014-bosc-biopython</a>
          </p>
        </section>

			</div>
      <img id="logo-bottom-left" src="img/bosc.png" />
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.min.js"></script>

		<script>

			// Full list of configuration options available here:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
        width: 1366 * 0.8,
        height: 768 * 0.6,
				controls: true,
				progress: true,
				history: true,
				center: true,
        slideNumber: true,
        previewLinks: true,

				theme: 'night' || Reveal.getQueryHash().theme,
				transition: Reveal.getQueryHash().transition || 'linear',

				// Parallax scrolling
				parallaxBackgroundImage: 'img/bg.jpg',
				parallaxBackgroundSize: '2170px 820px',

				// Optional libraries used to extend on reveal.js
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
					{ src: 'plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } }
				]
			});

		</script>

	</body>
</html>
